package.json

```diff
@@ -53,9 +54,10 @@
         "webpack-cli": "^3.3.12"
     },
     "dependencies": {
+        "extract-zip": "^2.0.1",
+        "lodash": "^4.17.19",
```

Comment:
Could you please clarify why we need to add lodash here? I couldn't find where it is used, and I remember we had problems with this dependency in the past.